src/mlagility/analysis/util.py

```diff
@@ -1,16 +1,12 @@
-import os
```

Comment:
Now that we have the clean command, should the cli arg`--lean-cache` become `--clean-cache`?

Reply:
Filed https://github.com/groq/mlagility/issues/150